src/App.tsx

```diff
@@ -15,7 +16,8 @@ function App() {
         <Routes>
           <Route path="/members" element={<MembersPage />} />
           <Route path="/members/:id" element={<MemberDetailsPage />} />
-          <Route path="/teams" element={<TeamsPage />} />
+          <Route path="/club_lead_teams" element={<LeadTeamsPage />} />
```

Comment:
nit: i don't like underscores in ur path, would prefer something like /teams/leadership and /teams/members or similar